### DISCUSSION TITLE
pushkin install version

### DISCUSSION THREAD

**Discussion by tmintz** [l3atbc-datadog/pushkin#91] [General] [answered]
Hi.  The quickstart guide recommends v.1.1.0, but the pushkin install package recommends v1.1.1.  Which should we use?  Thatnks!

**Comment by conbainbridge** [comment, marked as answer]
Apologies, we haven't updated the quickstart guide just yet! v1.1.1 is best to use.

**Reply by tmintz** [reply]
Thanks!
